hnn_core/param/default.json

```diff
@@ -15,9 +15,9 @@
     "prng_seedcore_input_dist": 2,
     "prng_seedcore_extpois": 2,
     "prng_seedcore_extgauss": 2,
-    "prng_seedcore_evprox_1": 2,
-    "prng_seedcore_evdist_1": 2,
-    "prng_seedcore_evprox_2": 2,
+    "prng_seedcore_evprox_1": -16,
+    "prng_seedcore_evdist_1": -16,
+    "prng_seedcore_evprox_2": -16,
```

Comment:
This was a hack to get the current seedcore tests in test_network.py to pass. We can change this once we feel comfortable with the new `net.gid_range` behavior.

Reply:
How do we want to handle this? Update the ground truth `dpl.txt` file?

Reply:
cc @jasmainak

Reply:
why exactly was this needed when new drive cells were added? can you remind me of the logic that leads to this new number?

Reply:
Originally, the 2 bursty drives were represented by 2 `_ArtificialCells`/gids, each with bursts of 10 "repeats". Now, each "repeat" corresponds to its own `_ArtificialCell`/gid for a total of 20 bursty drive gids (or 18 new gids).

I've adjusted the `prng_seedcore` values to maintain their original impact by subtracting the upward shift in gid values: 2-18=-16.

Reply:
I wouldn't touch this file. Even if you fix it for this file, it will be "broken" for user param files. Either you adjust it internally or just update the ground truth file as done before ...

Reply:
I agree - I put this here only to demonstrate that my changes to the gid assignment in `gid_ranges` could still pass our original tests with only a slight modification to the seedcore values.